meson.build

```diff
@@ -889,7 +889,10 @@ nobody_user = get_option('nobody-user')
 nobody_group = get_option('nobody-group')
 
 if not meson.is_cross_build()
-        getent_result = run_command('getent', 'passwd', '65534', check : false)
+        find_getent_result = find_program('getent', required : false)
+        getent_result = find_getent_result.found() ? \
+                run_command('getent', 'passwd', '65534', check : false) : \
+                run_command('false')
```

Comment:
just add one more nesting rather than running 'false' - shelling slows down the build